README.MD

```diff
@@ -1,57 +1,133 @@
 
-# REACT-Template
 
-> Starter template with linters and bable setup for react projects
+<a name="readme-top"></a>
 
+# 📗 Table of Contents
 
-# Note:
-this is a usage guide, remove it once you install all requirements
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
+<!-- PROJECT DESCRIPTION -->
 
+# 📖 [School Library] <a name="about-project"></a>
+
+> This project is built to manage school library
+> **[your_project__name]** School library
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```

Comment:
- [ ] [OPTIONAL] These instructions should be removed, they just there to give guide.